openmeter/billing/customeroverride.go

```diff
@@ -117,3 +117,42 @@ type GetProfileWithCustomerOverrideInput namespacedCustomerID
 func (g GetProfileWithCustomerOverrideInput) Validate() error {
 	return namespacedCustomerID(g).Validate()
 }
+
+type GetCustomerOverrideAdapterInput struct {
+	Namespace  string
+	CustomerID string
```

Comment:
should this be `customerentity.CustomerID`? Which  is basically namespace + customerId and has a Validate method